packages/beacon-node/src/network/subnets/attnetsService.ts

```diff
@@ -368,10 +367,13 @@ export class AttnetsService implements IAttnetsService {
     metrics.attnetsService.subscriptionsCommittee.set(this.shortLivedSubscriptions.size);
     // track short lived subnet status, >= 6 (Dlo) means healthy, otherwise unhealthy
     const currentSlot = this.clock.currentSlot;
+    const epoch = computeEpochAtSlot(currentSlot);
+    const fork = this.config.getForkInfoAtEpoch(epoch).name;
```

Comment:
is this only here because we will need epoch later? right now (changes in this PR) it seems unnecessary to calculate it

Reply:
Yea we need epoch later. Can revert this change in this PR and put it in the next PR